redash_dynamic_query/__init__.py

```diff
@@ -1,5 +1,6 @@
 import time
 
+import pystache
```

Comment:
The reason why I choose pystache, It's because Redash uses pystache currently.

Reply:
👌